packages/widgetbook/test/src/addons/utils/theme_wrapper.dart

```diff
@@ -75,55 +54,62 @@ Future<void> ensureCorrectThemeIsRendered<T>({
   expect(coloredBox.color, equals(colorBlue));
 }
 
-Future<void> ensureMultipleThemesRenderedSideBySide<T>({
+Future<void> renderAddonSideBySide<T>({
   required WidgetTester tester,
   required Widget sut,
   required List<WidgetbookAddOn> addons,
-  WidgetbookTheme<T>? updateTheme,
-  bool checkSingleTheme = false,
+  required List Function(Finder)? itemsCollection,
+  required List? expectedValues,
+  required Key itemsKey,
+  Future Function(BuildContext T)? updateAddon,
+  bool shouldUpdate = false,
```

Comment:
`shouldUpdate` and a having a nullable function `updateAddon` is a bit redundant. If you don't want to update, you don't provide the function.